src/modules/LanItems.rb

```diff
@@ -1407,9 +1407,11 @@ def BuildLanOverview
 
         mac_dev = HTML.Bold("MAC : ") + item_hwinfo["mac"].to_s + "<br>"
         bus_id  = HTML.Bold("BusID : ") + item_hwinfo["busid"].to_s + "<br>"
+        physical_port_id = HTML.Bold("PhysicalPortID : ") + physical_port_id(ifcfg_name).to_s + "<br>"
```

Comment:
Because i modified the physical_port_id but didn't remove this defensive call. So thnx for comment it will be fixed in next commit ;)